docs/source/tutorials/07_using_model_hub_nodes.rst

```diff
@@ -0,0 +1,334 @@
+.. include:: /include/substitution.rst
+
+*********************
+Using Model Hub Nodes
+*********************
+
+PeekingDuck has support for external model hub models. You can leverage the model
+hub nodes to use these external models with the PeekingDuck pipeline. The model
+hub nodes differ in design from traditional model nodes as the model hub models
+perform a variety of computer vision tasks. This tutorial demonstrates how
+to use these nodes through some sample pipelines.
+
+
+List of Model Hub Nodes
+=======================
+
+The table below shows the model hub nodes available.
+
++-----------------------------------------------------------------+---------------------------------+
+| Model                                                           | Documentation                   |
++=================================================================+=================================+
+| `Hugging Face Hub <https://huggingface.co/docs/hub/index>`_     | :mod:`model.huggingface_hub`    |
++-----------------------------------------------------------------+---------------------------------+
+| `MediaPipe Solutions <https://google.github.io/mediapipe/>`_    | :mod:`model.mediapipe_hub`      |
++-----------------------------------------------------------------+---------------------------------+
+
+
+Hugging Face Hub
+================
+
+The :mod:`model.huggingface_hub` node supports transformer models which perform
+the following computer vision tasks:
+
+   #. Instance segmentation
+   #. Object detection
+
+You can use the following command to get a list of supported computer vision tasks:
+
+   .. admonition:: Terminal Session
+   
+      | \ :blue:`[~user/project_dir]` \ > \ :green:`peekingduck model-hub huggingface tasks` \
+      | Supported computer vision tasks: ['instance segmentation', 'object detection']
+
+
+Instance Segmentation
+---------------------
+
+This example shows how Hugging Face Hub's instance segmentation models can be used
+to blur computer related objects from `cat_and_computer.mp4 
+<https://storage.googleapis.com/peekingduck/videos/cat_and_computer.mp4>`_.
+
+Supported Models
+^^^^^^^^^^^^^^^^
+
+To get a list of supported instance segmentation models, use the following CLI command:
+
+   .. admonition:: Terminal Session
+   
+      | \ :blue:`[~user/project_dir]` \ > \ :green:`peekingduck model-hub huggingface models -\-task 'instance segmentation'` \
+      | Supported hugging face \`instance_segmentation\` models:
+      | facebook/detr-resnet-101-panoptic
+      | facebook/detr-resnet-50-dc5-panoptic
+      | facebook/detr-resnet-50-panoptic
+      | facebook/maskformer-swin-tiny-coco
+      |
+      | <omitted>
```

Comment:
Suggest rephrasing `<omitted>` as `<long list truncated>`, as `<omitted>` alone might be misconstrued as part of the output.

Reply:
Changed